lib/_http_common.js

```diff
@@ -195,3 +195,11 @@ function httpSocketSetup(socket) {
   socket.on('drain', ondrain);
 }
 exports.httpSocketSetup = httpSocketSetup;
+
+/** Verifies that the given val is a valid HTTP token
```

Comment:
All lines should be prepended with `*`.  If there's a comment on the first line please only use one `*`.